### ISSUE TITLE
dev builds url

### ISSUE THREAD

**Issue by ziceptor** [trentech/WalletDrop#15]
All (all projects) dev builds url not work.  Timeout has expired.

**Comment by trentech** [comment]
Yeah my router died. I haven't updated the DNS records yet. Should be back this weekend
